### DISCUSSION TITLE
29/04

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#84] [Daily Sprint]
Que hice ayer?
Que estoy haciendo?
Algún impedimento?

**Comment by Pamela198713** [comment]
Ayer terminé de revisar el repositorio para ver qué no falte nada.
Hoy participe de una clase por fuera del ispc.
Impedimento: tiempo y organización

**Comment by rpgrca** [comment]
Ayer ayudé creando algunas tareas para el sprint 1. Mañana tengo que ir al trabajo presencial pero voy a estar a tiempo para la clase. También voy a empezar a armar algunos ejemplos de Postman para las pruebas y tal vez una al menos de Selenium, y luego empezar con lo de Android para ganar tiempo.

**Comment by fmdcba** [comment]
Ayer estuve revisando la documentación que se preparó del ieee830 y finiquitado detalles de organización del repo. Hoy repasé las tecnologías del semestre para aportar a las issues y pienso iniciar la investigación para emprendurismo + tecnología y desarrollo cuanto antes
